### ISSUE TITLE
Understand what's going on with big leaves

### ISSUE THREAD

**Issue by wcornwell** [KarinaGuo/Hons#9]
There has to be an upper limit to this method--no way we could measure banana leaf sizes from herbarium sheets for example. We just have to figure out what that is in practice.

**Comment by KarinaGuo** [comment]
Was just talking to Jason about this, for now I think the solution is just to be upfront in our discussion that it's a limitation to using herbarium sheets. Finding the upper limit would be another set of testing. I'd imagine it'd be a distribution; the larger the leaves the less it's detected using our model. It's essentially shown in the graph we were talking about yesterday, the correlation between smaller leaves and traditionally measured leaves is quite strong, but trails off when we get bigger.

![image](https://user-images.githubusercontent.com/100897883/215921475-dd752e94-00c4-45e3-a687-854bb9df3e57.png)

A potential workflow I could get on working with, let me know what you think:

- Gather a list of sheets ranging from species of small leaves to large leaves (total ~100 sheets, selected based on AusTraits mean leaf size based on species)
- Use the model to predict onto this list of sheets
- Draw up a table with the columns: Species, AusTrait's mean area, ML mean area, number of valid leaves detected (TP), number of invalid leaves detected (FP), number of leaves missed (FN), visual notes of sheet from a strict category (i.e., overlapping, damaged, etc.); with the latter 4 columns manually filled.

**Comment by wcornwell** [comment]
I think i agree. 

We do seem to detect leaves up to 100 cm2 but we have no species means above 16cm2.

it could be that because of the constraint of the size of the herbarium sheet, errors for the big leaved species are always smaller than the true values which means our species means are getting biased too small for those species.  

I guess the options are:

1. Exclude big leaved species 
2. Try extra hard to find the errors because we know about this bias
3. Exclude all of the smallest leaves for all species (Karina found a paper that did this) 
4. Exclude the smallest leaves only for big leaved species because of this asymmetric error issue. 

Hmmm

**Comment by KarinaGuo** [comment]
A few more plots on this point, was chatting with Jason yesterday and he mentioned doing a sd v mean scatter plot, where red is the linear relationship and blue is a loess line. We see that it starts to plateau around ~13cm2 (not logged). Possibly we could use this as a cut-off point and exclude big-leaved species, mentioning that there were too much error in our dataset for these leaves. From here, another alternative we considered was using extrapolation, or something of the sort, based on the natural distribution of leaf sizes. However, they seem to vary between species, I've popped the .pdf into the separate directory too

![image](https://user-images.githubusercontent.com/100897883/216208904-45cf491b-c9b7-46d1-aba8-21fb1c0e3e8f.png)

Another thing we wanted to look at was seeing whether the classifier had a bias of what it was _kicking out_, so I made the table mentioned above. I've put the .rmd and the related data files into a separate directory in this repo. A few things I noticed was that, our model did a good job at picking up all the valid leaves that were available, even in sheets with big leaves. The main factor for the paucity of big-leaf data points is the herbarium sheets, for example the one below.

![image](https://user-images.githubusercontent.com/100897883/216209618-5c434dc7-fc67-40bc-94ce-4cabc90eb26e.png)
